tests/conftest.py

```diff
@@ -1382,6 +1414,267 @@ def nodes():
     return nodes
 
 
+@pytest.fixture()
+def uploaded_objects(request, mcg_obj, awscli_pod):
+    return uploaded_objects_fixture(request, mcg_obj, awscli_pod)
+
+
+@pytest.fixture(scope='session')
+def uploaded_objects_session(request, mcg_obj_session, awscli_pod_session):
+    return uploaded_objects_fixture(
+        request,
+        mcg_obj_session,
+        awscli_pod_session
+    )
+
+
+def uploaded_objects_fixture(request, mcg_obj, awscli_pod):
+    """
```

Comment:
Please add empty lines after each section in the docstring

Reply:
Done